Tasks/Common/azure-arm-rest/azure-arm-common.ts

```diff
@@ -49,11 +55,20 @@ export class ApplicationTokenCredentials {
         this.authorityUrl = authorityUrl;
         this.activeDirectoryResourceId = activeDirectoryResourceId;
         this.isAzureStackEnvironment = isAzureStackEnvironment;
+        this.scheme = AzureModels.Scheme[scheme];
```

Comment:
If there is no scheme, do you want to initialize it as SPN?

Reply:
It will automatically goes into SPN case if there is no scheme But I will initialize it with SPN.